snippets/csharp/VS_Snippets_CLR/string.compare4/CS/comp4.cs

```diff
@@ -1,4 +1,4 @@
-﻿//<snippet1>
+//<snippet1>
```

Comment:
@WilliamAntonRohm Is this one supposed to be interactive? If so, the `<snippet1>` and `</snippet1`> markers should move to inside `Main`. (I can't tell from just this PR.)

Reply:
@BillWagner this sample is not interactive, as it relies on culture settings.